ParkingZone/Properties/serviceDependencies.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
why did you added this files? I saw here is the connection string but project is so big to separate this file?